envconfig_test.go

```diff
@@ -70,6 +75,7 @@ type EmbeddedButIgnored struct {
 
 func TestProcess(t *testing.T) {
 	var s Specification
+	s.NestedInterface = &Anonymous{}
```

Comment:
Since you have to do this anyway, why wouldn't a user just define the struct field as an `Anonymous`?